remix/app/repositories/newsletter.server.test.ts

```diff
@@ -26,4 +26,32 @@ describe('Newsletter Repository', () => {
       );
     });
   });
+
+  describe('findOne', () => {
+    it('returns a newsletter', async () => {
+      const newsletter = { ...newsletterFactory };
+
+      prismaMock.newsletter.findUnique.mockResolvedValue(newsletter);
+
+      await expect(
+        NewsletterRepository.findOne({ where: { id: newsletter.id } })
+      ).resolves.toEqual(newsletter);
+    });
+  });
+
+  describe('update', () => {
+    it('updates a newsletter', async () => {
+      const result = { count: 1 };
```

Comment:
You forgot to remove it following the last update 🙃

Reply:
Oh, for that one I keep it for the mock update on line 47 😬 or I should write `{count: 1}` on the mock too.

Reply:
Yes. It is okay to repeat code in tests. It is part of the best practice not to not over-engineering tests (see https://github.com/nimblehq/compass/pull/1274/files#diff-bc1c8aea77bf01da18ac0ba6c224d40694f9c88fa9a6f7d2a574567372ae884cR101)

Reply:
Yep, Updated here: [44c2663](https://github.com/nimblehq/js-framework-benchmark/pull/126/commits/44c2663cca727141d796f419d81b3b53cc5e7786)